### ISSUE TITLE
Add the ability to configure the Vector SDK from the VectorCloud

### ISSUE THREAD

**Issue by krconv** [rmountjoy92/VectorCloud#11]
Is there a way that we could configure the Vector SDK directly from VectorCloud? Maybe on the settings page, we could scan the network for Vector, and input the information needed for the SDK? 

I want to be able to run VectorCloud all the time. That way, I can host it on another computer and control Vector whenever I want.

**Comment by rmountjoy92** [comment]
I have this mostly set up already but I am limited to the settings in the sdk_config.ini in the home folder, you'll notice if your Vector's IP address changes a page will pop up where you can change the IP address. Unfortunately with the SDK right now I can only do: select which Vector serial number to use, change IP, display name and IP from config file. I could make edits to Anki's config.py script so people could download the cert file as well, but other than that I don't see what else I could do. I really want the ability to change network settings, eye color, volume, name, etc, but Anki hasnt made them available to the SDK yet

**Comment by krconv** [comment]
![image](https://user-images.githubusercontent.com/1918630/50899452-2d974080-13e0-11e9-80bf-fd7df22c851a.png)
Just tried out the IP change feature, that seems really useful!

I think that something that could be useful is making VectorCloud an application can just be running all the time (independent of whether there is a connected Vector), where you can configure new Vector robots, and run scripts on any Vector it has found and connected to. I guess the way I see that happening is making VectorCloud manage a list of Vectors. It might start up with no Vectors the first time, or it might be connected to multiple Vectors at once. The VectorCloud service itself could be configured, and then each Vector could also be individually added, removed, or configured (once the SDK allows).

**Comment by rmountjoy92** [comment]
This is great idea and I've played around with something like this in my head, but it will take a decent amount of shifting stuff around. The main obstacle is that nearly every page relies on a decent amount of data from Vector to populate the interface, so I would need to make a graceful way to deal with that lack of data, i.e. hiding icons that wouldn't work, etc. I will officially put this on the roadmap :)

p.s. For now, I recently added support for multiple Vectors, if you go to the settings page there's a link to select which serial to use.
